README.md

```diff
@@ -220,7 +220,7 @@ and
 
 ```js
 [
-  [rootMetafeedId, null],
+  [rootMetafeedId, rootDetails],
```

Comment:
changed this API to match the `rootDetails` behaviour with `findAndCreate` (i.e. give back actual details).
Needed this for writing simpler tests (no `null` edge cases, can always ask the purpose)

Reply:
ok making this another PR

Reply:
MOVED :heart: